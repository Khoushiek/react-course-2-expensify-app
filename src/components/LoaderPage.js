import React from "react";

const LoaderPage = () => (
  <div className="loader">
    <img className="loader__image" src="/images/loader1.gif" />
  </div>
);

export default LoaderPage;