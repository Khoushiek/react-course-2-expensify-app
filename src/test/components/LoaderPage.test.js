import React from "react";
import LoaderPage from "../../components/LoaderPage";
import {shallow} from "enzyme";

test("should return the Loader page component correctly", () => {
  const wrapper = shallow(<LoaderPage/>);
  expect(wrapper).toMatchSnapshot();
});