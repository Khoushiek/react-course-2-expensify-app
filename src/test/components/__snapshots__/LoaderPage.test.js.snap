// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the Loader page component correctly 1`] = `
<div
  className="loader"
>
  <img
    className="loader__image"
    src="/images/loader1.gif"
  />
</div>
`;
